asr-noise-augmentation.ipynb

```diff
@@ -0,0 +1,605 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"http://developer.download.nvidia.com/notebooks/dlsw-notebooks/rivaasrasr-noise-augmentation/nvidia_logo.png\" style=\"width: 90px; float: right;\">\n",
+    "\n",
+    "# How to improve accuracy on noisy speech by fine-tuning the Acoustic Model (Conformer-CTC) in the Riva ASR pipeline \n",
+    "\n",
+    "This tutorial walks you through some of the advanced customization features of the Riva ASR pipeline by fine-tuning the Acoustic Model (Conformer-CTC). These customization features improve accuracy on specific speech scenarios, like background noise and different acoustic environments."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## NVIDIA Riva Overview\n",
+    "\n",
+    "NVIDIA Riva is a GPU-accelerated SDK for building Speech AI applications that are customized for your use case and deliver real-time performance. <br/>\n",
+    "Riva offers a rich set of speech and natural language understanding services such as:\n",
+    "\n",
+    "- Automated speech recognition (ASR)\n",
+    "- Text-to-Speech synthesis (TTS)\n",
+    "- A collection of natural language processing (NLP) services, such as named entity recognition (NER), punctuation, and intent classification.\n",
+    "\n",
+    "In this tutorial, we will show how to augment your training data with background noise data for fine-tuning the Acoustic Model (Conformer-CTC) to improve accuracy on audio with background noise.  \n",
+    "To understand the basics of Riva ASR APIs, refer to [Getting started with Riva ASR in Python](https://github.com/nvidia-riva/tutorials/blob/stable/asr-python-basics.ipynb). <br>\n",
+    "\n",
+    "For more information about Riva, refer to the Riva [product page](https://developer.nvidia.com/riva) and [documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/overview.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data preprocessing\n",
+    "\n",
+    "For fine-tuning we need audio data with background noise. If you already have such data, then you can use it directly.  \n",
+    "In this tutorial, we will take the AN4 dataset and augment it with noise data from the Room Impulse Response and Noise Database from the [openslr database](https://www.openslr.org/28/).\n",
+    "\n",
+    "In this tutorial, we will be using NVIDIA NeMo for the data preprocessing step.\n",
+    "\n",
+    "#### NVIDIA NeMo Overview\n",
+    "\n",
+    "NVIDIA NeMo is a toolkit for building new state-of-the-art conversational AI models. NeMo has separate collections for Automatic Speech Recognition (ASR), Natural Language Processing (NLP), and Text-to-Speech (TTS) models. Each collection consists of prebuilt modules that include everything needed to train on your data. Every module can easily be customized, extended, and composed to create new conversational AI model architectures.\n",
```

Comment:
"...train on your data" seems odd to me. You train your model with your data.  What are your training on your data?

Reply:
I changed "train on your data" to "train conversational AI models on your data". See https://github.com/nvidia-riva/tutorials/pull/120/commits/2d4d8b85ef588889e1238241b0f93d0c3e9ab996.